packages/@vue/cli/bin/vue.js

```diff
@@ -57,6 +57,10 @@ program
   .option('-b, --bare', 'Scaffold project without beginner instructions')
```

Comment:
please adjust the indent (L57-L69) ...

Reply:
Modified the if statement (introduced space in between the `if` keyword and parentheses)